util/fine_tune/fine_tune_stats.py

```diff
@@ -0,0 +1,22 @@
+import typer
+import subprocess
+
+
+def fine_tune(openai_api_key: str, results_file_id: str, output_file: str):
+    # Check if required inputs are provided
+    if not (openai_api_key and results_file_id and output_file):
```

Comment:
I think typer will throw this error automatically so you don't need to check yourself, but I might be wrong.